.env.example

```diff
@@ -27,14 +27,14 @@ PLAID_PRODUCTS=auth,transactions
 # See https://plaid.com/docs/api/tokens/#link-token-create-request-country-codes for a complete list
 PLAID_COUNTRY_CODES=US,CA
 
-# Only required for OAuth:
-# For sandbox, set PLAID_REDIRECT_URI to 'http://localhost:3000/'
+# PLAID_REDIRECT_URI is an optional Quickstart setting for advanced users only
```

Comment:
This seems a little too far in the other direction -- don't we still want to encourage people to use it for Chase? Maybe remove "for advanced users only"?

Reply:
So we definitely want people to use redirect uris for real apps, but I guess I don't think people will be testing app-to-app using the Quickstart because the Quickstart doesn't have a mobile SDK frontend -- it seems like the only reasonable scenarios for the quickstart are desktop web or maybe mobile web, and even then mobile web now only cares about redirect uris if you're viewing mobile web in a webview, which again seems pretty unlikely for the quickstart. Or did I miss your point? (Like, do you think this sends the message that you shouldn't use redirect uris in a real app either?)

Reply:
Yeah, that was my concern. I kinda read this and thought, "Oh, maybe we don't want people to use Redirect URLs anymore", not "We don't want people to worry about Redirect URLs in the Quickstart"

Reply:
```suggestion
# PLAID_REDIRECT_URI is optional for this Quickstart application.
```